src/Renderer/TwigRenderer.php

```diff
@@ -13,6 +13,16 @@
  */
 class TwigRenderer
 {
+    private $twig_template_name = 'index.html.twig';
+    private $twig_template_path = __DIR__ . '/../../templates';
+
+    public function setRenderOptions(array $render_options)
+    {
+        foreach ($render_options as $render_option_name => $render_option_value) {
+            $this->$render_option_name = $render_option_value;
```

Comment:
Does this work?  I've not seen $this->$render before.  where you have a variable as a reference to a $this, does it need to be $this->{$render_option_name}

Reply:
It does work. I have used that often and never saw the { } syntax in my old legacy basis before. Probably this whole magic here is bad.